mbf_msgs/action/ExePath.action

```diff
@@ -30,6 +30,10 @@ uint8 TF_ERROR        = 111
 uint8 NOT_INITIALIZED = 112
 uint8 INVALID_PLUGIN  = 113
 uint8 INTERNAL_ERROR  = 114
+uint8 STOPPED         = 115
```

Comment:
Add comment as in planner?
Also, I like STOPPED as the last code,  just personal preference